hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/DeploymentManagementTest.java

```diff
@@ -1016,6 +1018,33 @@ public void forceAlreadyForcedActionNothingChanges() {
         findAction = deploymentManagement.findAction(actionId).get();
         assertThat(findAction.getActionType()).as("action type is wrong").isEqualTo(ActionType.FORCED);
     }
+    
+    @Test
```

Comment:
maybe move to DistributionSetManagementTest (verify assignemnt doesn't contain....)